hack/make-rules/test-integration.sh

```diff
@@ -41,27 +41,33 @@ KUBE_TEST_ARGS=${KUBE_TEST_ARGS:-}
 kube::test::find_integration_test_dirs() {
   (
```

Comment:
I agree that this script could use some cleanup, but for now I only have time to make it consistent.